routing.rst

```diff
@@ -475,6 +475,13 @@ that are special: each adds a unique piece of functionality inside your applicat
 ``_format``
     Used to set the request format (:ref:`read more <routing-format-param>`).
 
+``_fragment``
```

Comment:
IMO, this is the wrong place for it, as you won't use ``_fragment`` in route patterns.

It should rather be explained in a section near ``Generating URLs with Query Strings``